alpine/release.go

```diff
@@ -21,81 +28,33 @@ const (
 	V3_3  Release = "v3.3"
 )
 
-// ReleaseToPrettyName maps a Release to the PrettyName found in alpine's os-release file.
-//
-// Official Alpine images consistantly have a Pretty_Name field in their os-release files.
-var ReleaseToPrettyName = map[Release]string{
-	V3_10: "Alpine Linux v3.10",
-	V3_9:  "Alpine Linux v3.9",
-	V3_8:  "Alpine Linux v3.8",
-	V3_7:  "Alpine Linux v3.7",
-	V3_6:  "Alpine Linux v3.6",
-	V3_5:  "Alpine Linux v3.5",
-	V3_4:  "Alpine Linux v3.4",
-	V3_3:  "Alpine Linux v3.3",
-}
-
 // Common os-release fields applicable for *claircore.Distribution usage.
 const (
 	Name = "Alpine Linux"
 	ID   = "alpine"
 )
 
-var alpine3_3Dist = &claircore.Distribution{
-	Name:       "Alpine Linux",
-	DID:        "alpine",
-	VersionID:  "3.3",
-	PrettyName: "Alpine Linux v3.3",
-}
-
-var alpine3_4Dist = &claircore.Distribution{
-	Name:       "Alpine Linux",
-	DID:        "alpine",
-	VersionID:  "3.4",
-	PrettyName: "Alpine Linux v3.4",
-}
-
-var alpine3_5Dist = &claircore.Distribution{
-	Name:       "Alpine Linux",
-	DID:        "alpine",
-	VersionID:  "3.5",
-	PrettyName: "Alpine Linux v3.5",
-}
-
-var alpine3_6Dist = &claircore.Distribution{
-	Name:       "Alpine Linux",
-	DID:        "alpine",
-	VersionID:  "3.6",
-	PrettyName: "Alpine Linux v3.6",
-}
-
-var alpine3_7Dist = &claircore.Distribution{
-	Name:       "Alpine Linux",
-	DID:        "alpine",
-	VersionID:  "3.7",
-	PrettyName: "Alpine Linux v3.7",
-}
-
-var alpine3_8Dist = &claircore.Distribution{
-	Name:       "Alpine Linux",
-	DID:        "alpine",
-	VersionID:  "3.8",
-	PrettyName: "Alpine Linux v3.8",
-}
-
-var alpine3_9Dist = &claircore.Distribution{
-	Name:       "Alpine Linux",
-	DID:        "alpine",
-	VersionID:  "3.9",
-	PrettyName: "Alpine Linux v3.9",
+func mkdist(maj, min int) *claircore.Distribution {
```

Comment:
have you confirmed that v3.12 and v3.11 follow the same format as all the others by looking at an os-release file in their official container release? often there are differences between version, just want to confirm.

Reply:
Yeah, via `podman run --rm docker.io/library/alpine:3.$v cat /etc/os-release /etc/issue`